src/Psalm/Config.php

```diff
@@ -194,6 +194,13 @@ class Config
      */
     public $use_docblock_property_types = false;
 
+    /**
+     * Whether using @property in docblocks should implicitly seal properties
```

Comment:
Psalm seem to parse `@property` in your docblock as a property. Maybe replace that with `using the property annotation should...`

Reply:
Haha... Ok, makes sense.

```suggestion
     * Whether using property annotations in docblocks should implicitly seal properties
```